easybuild/easyconfigs/q/Qiskit/Qiskit-0.10.1-foss-2019a.eb

```diff
@@ -0,0 +1,153 @@
+        'source_urls': ['https://github.com/Qiskit/qiskit-terra/archive/'],
+        'source_tmpl': '%(version)s.tar.gz',
+        'checksums': ['cbb52b27b8389fcb6a5c1f69f23f7366533d147d61babdb28c104ee106d8d91d'],
+        'preinstallopts': "python setup.py build_ext --inplace && ",
+        'modulename': 'qiskit.qobj',
+    }),
+    ('scikit-build', '0.9.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/s/scikit-build/'],
+        'checksums': ['81119231cd9c4eba7acc1aaf7351a33b69494c2ccf94a31c2902a80622f089a9'],
+        'modulename': 'skbuild',
+    }),
+    ('qiskit-aer', '0.2.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/q/qiskit-aer/'],
+        'checksums': ['00a663ca066eea50cd42b852241351bb178f9e4910a47ab2a4706cc3a49534d5'],
+        'modulename': 'qiskit.providers.aer',
+    }),
+    ('pyeda', '0.28.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pyeda/'],
+        'checksums': ['07185f458d5d0b2ba5058da8b95dad6ab7684ceaf41237a25bcd3f005490f59d'],
+    }),
+    ('more-itertools', '5.0.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/m/more-itertools/'],
+        'checksums': ['38a936c0a6d98a38bcc2d03fdaaedaba9f412879461dd2ceff8d37564d6522e4'],
+    }),
+    ('inflection', '0.3.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/i/inflection/'],
+        'checksums': ['18ea7fb7a7d152853386523def08736aa8c32636b047ade55f7578c4edeb16ca'],
+    }),
+    ('Quandl', '3.4.8', {
+        'source_urls': ['https://pypi.python.org/packages/source/Q/Quandl/'],
+        'checksums': ['63cdd98e192e85abc2e9a5f5bead243dc82b663bcb98894d379f4617c4b7339d'],
+    }),
+    ('fastdtw', '0.3.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/f/fastdtw/'],
+        'checksums': ['09a24d62a8eb72f87c03692dffae4b8d8499b53f503e9abce5e717c5b705dff9'],
+    }),
+    ('docplex', '2.9.141', {
+        'source_urls': ['https://pypi.python.org/packages/source/d/docplex/'],
+        'checksums': ['8bc4b4f073af18ffec139cf244bf58146b8be46fd45dfba88361c4c6ab68e313'],
+    }),
+    ('dlx', '1.0.4', {
+        'source_urls': ['https://pypi.python.org/packages/source/d/dlx/'],
+        'checksums': ['ef75bc9d590216ebde7d4811f9ae6b2d6c6dc2a54772d94ae13384dc517a5aae'],
+    }),
+    ('docloud', '1.0.369', {
+        'source_urls': ['https://pypi.python.org/packages/source/d/docloud/'],
+        'checksums': ['01cb439f5bbc6366f4fbb9a9185655301f645d3356476b6191199f334dd7a771'],
+    }),
+    ('qiskit-ignis', '0.1.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/q/qiskit-ignis/'],
+        'checksums': ['29af3f3dd8d485ad3d438713c3416f307bb743047f759039c28c789130028262'],
+    }),
+    ('qiskit-aqua', '0.5.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/q/qiskit-aqua/'],
+        'checksums': ['6302a6d1107a1c2b5ed06d5fc40b190743a0956cbe68b3d07d604b4ccd4aacc4'],
+    }),
+    ('pyscf', '1.6.1.post1', {
```

Comment:
This bit needs more work, should probably become a full dependency since it's non-trivial (only `.whl` available in PyPI)